python-sdk/pyproject.toml

```diff
@@ -48,7 +48,8 @@ tests = [
     "pytest-cov",
     "pytest-describe",
     "mypy",
-    "sqlalchemy-stubs"  # Change when sqlalchemy is upgraded https://docs.sqlalchemy.org/en/14/orm/extensions/mypy.html
+    "sqlalchemy-stubs",  # Change when sqlalchemy is upgraded https://docs.sqlalchemy.org/en/14/orm/extensions/mypy.html
+    "openlineage-airflow>=0.15.1"
```

Comment:
Why do we need to add `openlineage-airflow` here? it is already in `[all]` extra

Reply:
It failed in test complaining `openlineage-airflow` is not installed

Reply:
it shouldn't though - check https://github.com/astronomer/astro-sdk/blob/95c870d19c722fdfd110d59d979b53e6a5ce79ea/python-sdk/noxfile.py#L35